lib/auth/permissions.go

```diff
@@ -477,6 +478,7 @@ func roleSpecForProxy(clusterName string) types.RoleSpecV6 {
 				types.NewRule(types.KindClusterAuditConfig, services.RO()),
 				types.NewRule(types.KindClusterNetworkingConfig, services.RO()),
 				types.NewRule(types.KindSessionRecordingConfig, services.RO()),
+				types.NewRule(types.KindUIConfig, services.RO()),
```

Comment:
Note: This line can't be backported until v13. The backports will work without it.